airflow-core/src/airflow/plugins_manager.py

```diff
@@ -586,6 +592,14 @@ def get_plugin_info(attrs_to_dump: Iterable[str] | None = None) -> list[dict[str
                         {**d, "app": qualname(d["app"].__class__) if "app" in d else None}
                         for d in getattr(plugin, attr)
                     ]
+                elif attr == "fastapi_root_middlewares":
+                    info[attr] = [
+                        {
+                            **d,
```

Comment:
I don't think include args and kwargs here is worth it -- the  risk of exposing sensitive info is too great.

Reply:
```suggestion
                            "name": d.get("name"),
```

Reply:
Yes I hesitated about that too, we can remove it indeed.

Done thanks.